### ISSUE TITLE
FR: Distinguish accounts by their tailnet name

### ISSUE THREAD

**Issue by sefidel** [tailscale/tailscale#9911]
### What are you trying to do?

I'm trying to use the fast user switching feature with my personal account, and some organisation accounts that are associated with my GitHub account. After adding all of my accounts, I've noticed that they all appear exactly the same (`sefidel@github`), presumably because they're all from my GitHub account.

As a result, I am unable to distinguish between my personal tailnet and multiple organisation-managed tailnets.

### How should we solve this?

Show the name of the tailnet alongside the account information.

### What is the impact of not solving this?

Users have to click each account until they find the tailnet they're looking for.

### Anything else?

Minimal example shown below:

![image](https://github.com/tailscale/tailscale/assets/71049646/3b07829a-3bbd-43b9-af32-96cf79d78664)

**Comment by 456dev** [comment]
In the CLI, `tailscale switch` cannot switch between these profiles since the target name is the same as the current name

additionally, the sorting seems not deterministic, but I'm unsure if that affects anything, since the names are the same
```
$ tailscale switch --list
X@gmail.com *
X@gmail.com
$ tailscale switch --list
X@gmail.com
X@gmail.com *
$ tailscale switch X@gmail.com
Already on account "X@gmail.com"
```

**Comment by pReya** [comment]
Just show the used organization name in the context menu, this would already fix this problem.

But right now, the switcher is basically unusable :(

<img width="240" alt="Screenshot 2023-12-08 at 12 20 06" src="https://github.com/tailscale/tailscale/assets/4677417/58d13aed-0054-47ca-9183-be2823c185f8">

**Comment by samlinville** [comment]
We have fixed this bug in the latest unstable release (1.55), and will release it in our upcoming stable release (1.56) next week.

Between now and next week when 1.56 is out, you can download the unstable releases here: https://tailscale.com/kb/1083/install-unstable/ If you use the version of our Mac app that's downloaded from the App Store, I recommend using the TestFlight build.

**Comment by pReya** [comment]
1.56 has been released, so this ticket can be closed, right?

**Comment by sefidel** [comment]
Yep, can confirm that the issue is fixed in 1.56.
